misaligned.cpp

```diff
@@ -15,7 +17,12 @@ int printColorMap() {
 
 void testPrintColorMap() {
     std::cout << "\nPrint color map test\n"; 
+    int PrintColorMap = 2;
+    int i = PrintColorMap / 5;
+    int j = PrintColorMap % 5;
     int result = printColorMap();
+    assert(std::string(majorColor[i]) == ("White"));

```

Comment:
Let the asserts express the consumer's need. The consumer of our code can be another developer, a service engineer or the end-user. Let's evaluate how far these asserts will assure the customer's need.
In this case, let's imagine that it's the end-user. The user would want the printColorMap to give the color pairs correctly, without repetition, etc.
Good part: These asserts ensure that the strings are defined correctly. Ensures that `printColorMap` is using the right strings.
Gap: However, they miss the bug on line 12, which uses `i` to index both major and minor colors, instead of using `i` and `j`.
Can you think of any assert to address the gap and catch that bug?